contracts/serverless-v2/Relay.sol

```diff
@@ -891,25 +894,28 @@ contract Relay is
         if(block.timestamp > jobStartTime + OVERALL_TIMEOUT)
             revert RelayOverallTimeoutOver();
 
-        // adjusting currentRuns to last 127 bits of jobSubsId to prevent txn replay
-        uint256 jobId = _jobSubsId | (jobSubs.currentRuns + 1);
+        uint256 instanceCount = _jobId & ((1 << 127) - 1);
+        // note: instance count for the first output should be 1
+        if(instanceCount <= jobSubs.currentRuns)
```

Comment:
this won't work for the case 0.
Fix it by using `<` only. And at L912 by setting currentRuns = instanceCount + 1.

indexing is from 0 for individual instances of job.

Reply:
OK, I considered it to be starting from 1. 
nvm, pushed the update.